File: flutter/lib/domain/lobby/complete_lobby.dart
class CompleteLobby {
  final String tag;
  final String name;
  final String uid;
  final String username;
  final int creationTime;
  final bool isFavorite;
  final int players;

  CompleteLobby(
      {required this.tag,
      required this.name,
      required this.uid,
      required this.username,
      required this.creationTime,
      required this.isFavorite,
      required this.players});
}

File: jvm/ship-conquest/src/main/kotlin/com/example/shipconquest/domain/lobby/CompleteLobby.kt
package com.example.shipconquest.domain.lobby

/**
 * The [CompleteLobby] data class holds the data for the complete set of
 * information available for a lobby.
 */
data class CompleteLobby(
    val tag: String,
    val name: String,
    val uid: String,
    val username: String,
    val creationTime: Long,
    val isFavorite: Boolean,
    val playerCount: Int
)
